manim/animation/creation.py

```diff
@@ -291,6 +292,65 @@ def _set_default_config_from_length(self, vmobject: VMobject) -> None:
             self.lag_ratio = min(4.0 / length, 0.2)
 
 
+class Unwrite(Write):
+    """Simulate erasing by hand a :class:`~.Text` or a :class:`~.VMobject`.
+
+    Parameters
+    ----------
+    reverse : :class:`bool`
+        Set True to have the animation start erasing from the last submobject first.
+
+    Examples
+    --------
+
+    .. manim:: UnwriteReverseFalse
+
+        class UnwriteReverseFalse(Scene):
+            def construct(self):
+                text = Tex("Alice and Bob").scale(3)
+                self.add(text)
+                self.play(Unwrite(text))
+
+    .. manim :: UnwriteReverseTrue
+
+        class UnwriteReverseTrue(Scene):
+            def construct(self):
+                text = Tex("Alice and Bob").scale(3)
+                self.add(text)
+                self.play(Unwrite(text,reverse=True))
+
+    """
+
+    def __init__(
+        self,
+        vmobject: VMobject,
+        run_time: float = None,
+        lag_ratio: float = None,
+        rate_func: typing.Callable[[float], np.ndarray] = linear,
+        reverse: bool = False,
+        **kwargs,
+    ) -> None:
+
+        if not reverse:
+            vmobject.submobjects[0].submobjects = list(
+                reversed(vmobject.submobjects[0].submobjects)
+            )
```

Comment:
1. Why only the first submobject?
2. Modifying the animated Mobject will be unexpected to users, especially since indexing into Latex is already tricky. If it isn't possible to do it with the rate function is it at least possible to override `begin()` to reverse the mobjects and override `end()` to restore them?

Reply:
Thank you for pointing those out! I'll try to get get those problems addressed within the next few days.

Reply:
I've reimplemented the reversing to address both issues, but I'm running into a problem where running something like
```python
class UnwriteScene(Scene):
    def construct(self):
        text = VGroup(Tex("Hello").shift(LEFT),Tex("Bye").shift(RIGHT))
        text[0].set_color(RED)
        self.play(Write(text))
        self.play(Unwrite(text))
        self.play(Write(text))
```
returns [this](https://user-images.githubusercontent.com/10787002/111532530-209ed600-873c-11eb-83d4-3f3574f4f1e8.mp4) instead of finishing by writing back to the screen, as I intended. I've noticed that the same thing happens with `ShowCreation` and `Uncreate` when used in the same fashion. Is this intentional?